R/sumbetadiff.R

```diff
@@ -1,27 +1,36 @@
-##' Mode and Credible Interval Calculation for The Difference between Two Beta Distributions
+##' Mode and Credible Interval Calculation for The Difference between Two Beta Distributions.
```

Comment:
I would not put a full stop in the title of the documentation
```suggestion
##' Mode and Credible Interval Calculation for The Difference between Two Beta Distributions
```